packages/core/admin/ee/server/routes/review-workflows.js

```diff
@@ -2,111 +2,175 @@
 
 const { enableFeatureMiddleware } = require('./utils');
 
-module.exports = {
-  type: 'admin',
-  routes: [
-    // Review workflow
-    {
-      method: 'GET',
-      path: '/review-workflows/workflows',
-      handler: 'workflows.find',
-      config: {
-        middlewares: [enableFeatureMiddleware('review-workflows')],
-        policies: [
-          'admin::isAuthenticatedAdmin',
-          {
-            name: 'admin::hasPermissions',
-            config: {
-              actions: ['admin::review-workflows.read'],
+const CONTENT_API_PREFIX = 'strapi_workflows';
```

Comment:
we normally use kebab case for the urls, I would recommend doing the same here
```suggestion
const CONTENT_API_PREFIX = 'strapi-workflows';
```